saltyrtc/__init__.py

```diff
@@ -3,22 +3,29 @@
 """
 import itertools
 
-from .exception import *
-from .common import *
-from .message import *
-from .protocol import *
-from .client import *
-from .server import *
-from . import util
+# noinspection PyUnresolvedReferences
+from .exception import *  # noqa
+# noinspection PyUnresolvedReferences
+from .common import *  # noqa
+# noinspection PyUnresolvedReferences
+from .message import *  # noqa
+# noinspection PyUnresolvedReferences
+from .protocol import *  # noqa
+# noinspection PyUnresolvedReferences
+from .client import *  # noqa
+# noinspection PyUnresolvedReferences
+from .server import *  # noqa
+# noinspection PyUnresolvedReferences
+from .util import *  # noqa
```

Comment:
Are you sure you want to use star imports? There's a reason code checking tools don't like them.

It breaks static analysis against this package. Autocompleting `saltyrtc.<tab>` will now fail in a lot of tools. And it will be hard to prevent name collisions (e.g. if `client` and `server` would both export a class called `Error`).